src/main/java/art/heredium/domain/post/model/dto/request/PostHistorySearchRequest.java

```diff
@@ -0,0 +1,21 @@
+package art.heredium.domain.post.model.dto.request;
+
+import java.time.LocalDateTime;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Getter
+@Setter
+public class PostHistorySearchRequest {
+  @JsonProperty("modify_date_from")
+  private LocalDateTime modifyDateFrom;
+
+  @JsonProperty("modify_date_to")
+  private LocalDateTime modifyDateTo;
+
+  @JsonProperty("modify_user")
+  private String modifyUserEmail;
```

Comment:
```suggestion
  private String modifyUserEmailOrName;
```
Mình sẽ hỗ trợ search theo email / tên người sửa đổi.

Reply:
Tôi đã thêm: cf968216403c55a9445611f7390d882145db3feb